README.md

```diff
@@ -20,157 +20,7 @@ External Contributors can read [Getting Started with OpenAPI Specifications](htt
 
 ## Directory Structure
 
-The structure of the directory should strictly follow these rules:
-
-1. **Profile**: The profile folder contains the profiles' definition files. The profile definition targets for hybrid applications that could run on Azure Stack general availability versions and Azure Cloud.
-
-2. **Specification**: This folder is the root folder for all Specs (Management Plane and Data Plane) related documents.
-
-3. **{RP-Name} Folders** - each resource provider should have at least one separate folder.
-    > If multiple folders are required? It depends on the following considerations:
-    >
-    > - An RP folder leads to a separate SDK package. Is it expected to have separate SDK packages for different service/component entities?
-    > - Service/component entities in one folder share the same versioning cycle. Can service/component entities in one folder share the same version label, and upgrade together in the future?
-    > - Specification files and AutoRest configuration files in one RP folder are better to refer to files in the same RP folder. Note: Entity type definition that needs to be referred cross RP folders should be placed and maintained under the folder [**common-types**](https://github.com/Azure/azure-rest-api-specs#common-types).
-    > - For more considerations, you may consult the reviewer in API design review. To initiate the review, Please submit an [Azure SDK intake questionnaire](https://aka.ms/sdk-apex).
-
-    RP folders may contain resource manager or data plane TypeSpec "specs". TypeSpec is a language for describing cloud service APIs and generating other API description languages, client and service code, documentation, and other assets. Explore more by visiting the tutorial in the TypeSpec repo: [TypeSpec tutorial](http://aka.ms/cadlTutorial). You can also ask questions for providing feedback in the internal Teams channel [TypeSpec Discussion](https://teams.microsoft.com/l/channel/19%3a906c1efbbec54dc8949ac736633e6bdf%40thread.skype/Cadl%2520Discussion%2520%25F0%259F%2590%25AE?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).
-
-    For more information about the structure of TypeSpec files in the repo see [TypeSpec repo structure](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md).
-   
-5. **'resource-manager' and 'data-plane' Folders**: the RPs can put specs in one of two categories: `resource-manager` (for ARM resources) and `data-plane` (for everything else). There should be an AutoRest configuration file (`readme.md`) for the RP inside both of these folders when present.
```

Comment:
@JeffreyRichter this bullet no. 5, together with bullet no 3., say that directories like `specification/confidentialledger ` correspond to a Resource Provider. Do you disagree with this?

> **'resource-manager' and 'data-plane' Folders**: the RPs can put specs in one of two categories: `resource-manager` (for ARM resources) and `data-plane` (for everything else). There should be an AutoRest configuration file (`readme.md`) for the RP inside both of these folders when present.

Reply:
This is amazing, thank you @JeffreyRichter to take time to write this down. I'll let you know once I incorporate it into the document.

Reply:
@JeffreyRichter I rewrote the doc following your advice. Could you have a look? I did not follow exactly what you suggested, but I think there is already significant improvement in the document correctness and overall flow. I am happy to iterate in future PRs.

Also, I would love feedback from: @maririos, @rkmanda, @weshaggard, @mikeharder, @mikekistler.